src/main.py

```
@@ -0,0 +1,104 @@
+    print(f'Logged in as {bot.user} (ID: {bot.user.id})')
+    print('------')
+    # Load Cogs
+    await bot.load_extension('cogs.verification')
+    await bot.load_extension('cogs.events')
+    await bot.load_extension('cogs.admin')
+    
+    # Sync Slash Commands
+    try:
+        synced = await bot.tree.sync()
+        print(f"Synced {len(synced)} command(s)")
+    except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)